### PR TITLE
Update FCS-GX to release v0.5.4

### PR DESCRIPTION
**Pull request by etvedte**
Updating conda recipe files for FCS-GX release v0.5.4

### CODE REVIEW

**Comment by richard-burhans** [comment]
Since the version number is changing to 0.5.4, the build number should be reset to 0.

**Comment by etvedte** [comment]
@richard-burhans - thanks for the info. This is the time I'm updating so I can refine our documentation to include this. So I should change this and add new commit? What did your commit achieve?

Are we responsible for addressing the failed checks?

**Comment by etvedte** [comment]
@BiocondaBot please add label

File: recipes/ncbi-fcs-gx/0.5.0/build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+set -uex
+
+mkdir -vp ${PREFIX}/bin
+
+ls -l .
+
+$GCC --version
+$GCC -print-search-dirs
```
Comment (martin-g) [inline recipes/ncbi-fcs-gx/0.5.0/build.sh:9]: The debug statements above could be removed

File: recipes/ncbi-fcs-gx/0.5.0/build.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+mkdir -vp ${PREFIX}/bin
+
+ls -l .
+
+$GCC --version
+$GCC -print-search-dirs 
+
+#cd fcs-gx-0.4.0
+make VERBOSE=1
```
Comment (martin-g) [inline recipes/ncbi-fcs-gx/0.5.0/build.sh:12]:
```suggestion
make VERBOSE=1 -j ${CPU_COUNT}
```

File: recipes/ncbi-fcs-gx/0.5.0/build.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+$GCC --version
+$GCC -print-search-dirs 
+
+#cd fcs-gx-0.4.0
+make VERBOSE=1 
+
+cp $SRC_DIR/build/src/gx    ${PREFIX}/bin/
+cp $SRC_DIR/scripts/*      ${PREFIX}/bin/
+cp $SRC_DIR/make_gxdb/blast_names_mapping.tsv ${PREFIX}/bin/ 
+echo PREFIX: ${PREFIX}
```
Comment (martin-g) [inline recipes/ncbi-fcs-gx/0.5.0/build.sh:17]: Debug

File: recipes/ncbi-fcs-gx/0.5.0/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    - gzip >=1.5
+    - pv >=1.4.6
+    - rclone =1.61.1
+    - grep >=3.4
+
+build:
+  number: 3
+  skip: True  # [not linux or not x86_64]
+  run_exports:
+    - {{ pin_compatible('ncbi-fcs-gx', max_pin="0.5") }}
```
Comment (martin-g) [inline recipes/ncbi-fcs-gx/0.5.0/meta.yaml:32]:
```suggestion
    - {{ pin_compatible('ncbi-fcs-gx', max_pin="x.x") }}
```

File: recipes/ncbi-fcs-gx/meta.yaml [hunk 5]
```diff
@@ -26,10 +23,10 @@ requirements:
     - grep >=3.4
 
 build:
-  number: 3
+  number: 0
   skip: True  # [not linux or not x86_64]
   run_exports:
-    - {{ pin_compatible('ncbi-fcs-gx', max_pin="0.5") }}
+    - {{ pin_compatible('ncbi-fcs-gx', max_pin="0.4") }}
```
Comment (martin-g) [inline recipes/ncbi-fcs-gx/meta.yaml:29]:
```suggestion
    - {{ pin_compatible('ncbi-fcs-gx', max_pin="x.x") }}
```

**Comment by etvedte** [comment]
@martin-g - do these look OK?

**Review by martin-g — APPROVED** [review]

**Pull request merged**